api/v1/tetragon/tetragon.proto

```diff
@@ -95,6 +95,7 @@ message Process {
     uint32 refcnt = 13;
     Capabilities cap = 14;
     Namespaces ns = 15;
+    google.protobuf.UInt32Value ns_pid = 16;
```

Comment:
I am not really sure that we need to export `ns_pid` to the user events. Another alternative can be to add a new field in `ProcessInternal` that keeps the `NSPID` value and use this when you need to call `GetPodInfo`.

Reply:
yep, i think that's better